cmd/ui/src/store.ts

```diff
@@ -59,7 +62,12 @@ const loadState = () => {
     }
 };
 
-const saveState = (state: any) => {
+type PersistedState = {
+    auth: { sessionToken: string | null };
+    global: { view: { darkMode: boolean; notifications: string[] } };
```

Comment:
I think we can just ignore `notifications` here unless we had a specific use case for wanting to persist those?

Reply:
Not having this defined when loading from the persisted state would throw an undefined error. It would be better to untangle this a bit more and we aren't actually persisting notifications as we only save an empty array in the throttled save state but this avoids the app crashing without a bigger refactor.

Reply:
My understanding is that by leaving stuff undefined redux will automatically overlay a default state.

For example, in the `auth` property we only care about persisting `sessionToken` meanwhile there are plenty of other pieces of state that we track.

```
    isInitialized: boolean;
    loginLoading: boolean;
    loginError: any;
    updateExpiredPasswordLoading: boolean;
    updateExpiredPasswordError: any;
    acceptEULALoading: boolean;
    acceptEULAError: any;
    sessionToken: string | null;
    user: getSelfResponse | null;
    fedramp_eula_accepted: boolean;
```

Reply:
Aaaaand I was wrong. So we'll leave it as is!